contracts/VM.sol

```diff
@@ -13,10 +12,12 @@ abstract contract VM {
     uint256 constant FLAG_CT_STATICCALL = 0x02;
     uint256 constant FLAG_CT_VALUECALL = 0x03;
     uint256 constant FLAG_CT_MASK = 0x03;
-    uint256 constant FLAG_EXTENDED_COMMAND = 0x80;
-    uint256 constant FLAG_TUPLE_RETURN = 0x40;
+    uint256 constant FLAG_TUPLE_RETURN = 0x80;
+    uint256 constant FLAG_EXTENDED_COMMAND = 0x40;
+    uint256 constant FLAG_FALLBACK = 0x20;
```

Comment:
Since you're trying to retrieve the bit "2", the `FLAG_FALLBACK` should be 0x04 since in binary that is b100

A good reference to see this is that the `FLAG_CT_MASK` is 0x03 which in binary is b11 gives bits "0" and "1"

Reply:
my bad. I was viewing the endianness incorrectly. time to get some lunch!